code/modules/antagonists/ashwalker/ashwalker.dm

```diff
@@ -23,9 +23,12 @@
 /datum/antagonist/ashwalker/get_team()
 	return ashie_team
 
-/datum/antagonist/ashwalker/on_body_transfer(mob/living/old_body, mob/living/new_body)
+/datum/antagonist/ashwalker/on_body_removal(mob/living/old_body)
 	. = ..()
 	UnregisterSignal(old_body, COMSIG_MOB_EXAMINATE)
```

Comment:
These args are incorrect

`/datum/antagonist/proc/on_body_removal(datum/mind/mind, mob/living/old_body)`

`/datum/antagonist/ashwalker/on_body_removal(mob/living/old_body)`

Reply:
oh right, forgot to update that, thanks